ui/packages/components/src/EventDetails/EventDetails.tsx

```diff
@@ -48,6 +50,7 @@ export function EventDetails({
   onReplayEvent,
   selectedRunID,
   SendEventButton,
+  codeBlockActions = [],
```

Comment:
I would love feedback on the changes to `EventDetails` as this is a shared component and y'all might have ideas on extensibility/composability in the future in light of recent work!

Reply:
🤔 Yeah, if we start adding other actions to code blocks I wonder if it would be better to just pass JSX rather than button props. Passing action props enforces the design, but it's definitely less versatile